tsconfig.base.json

```diff
@@ -9,6 +9,8 @@
     "paths": {
       "foam-core": ["./packages/foam-core/src"],
       "foam-vscode": ["./packages/foam-vscode/src"]
-    }
+    },
+    "typeRoots": ["node_modules/@types"],
+    "sourceMap": true 
```

Comment:
What are these for?
is it specific to the move to web extension or did you uncover some issue with the current configuration?